### ISSUE TITLE
Folium loads two versions of bootstrap

### ISSUE THREAD

**Issue by bopjesvla** [python-visualization/folium#1908]
**Describe the bug**

Folium currently loads two versions of bootstrap:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
<link rel="stylesheet" href="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css"/>
```

**To Reproduce**

The first Folium iframe on https://python-visualization.github.io/folium/latest/getting_started.html contains the above snippet.

**Comment by Conengmo** [comment]
That's correct, we require Bootstrap 3 to show Glyphicons. See https://github.com/python-visualization/folium/pull/1650.

But I did just notice in https://stackoverflow.com/a/18225474/4082914 they also list a css just for Glyphicons. It seems to work as well. Not sure why we didn't include that in https://github.com/python-visualization/folium/pull/1650. I'll open a PR.
